mappings/net/minecraft/block/PitcherBlock.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/unmapped/C_fntbpikr net/minecraft/block/PitcherBlock
```

Comment:
```suggestion
CLASS net/minecraft/unmapped/C_fntbpikr net/minecraft/block/PitcherCropBlock
```
make sure to match the identifier!

Reply:
I named it like that because it was done the same way for the Torchflower
![image](https://user-images.githubusercontent.com/40738104/227513882-5702b605-3fe9-459a-882c-a56e4bdb96af.png)